vllm/model_executor/layers/attention.py

```diff
@@ -163,16 +165,20 @@ def forward(
             )
             output = out.view_as(query)
         else:
-            # Decoding run.
-            output = _paged_attention(
-                query,
-                key_cache,
-                value_cache,
-                input_metadata,
-                self.head_mapping,
-                self.scale,
-                self.alibi_slopes,
-            )
+            if FLAGS.ENABLE_SD:
```

Comment:
correct me if I'm wrong, but for assisted decoding, usually the propse_cnt is small (maybe around 4?), which would cause first dimension of `q` to be small, thus the `q@k` gemm and `qk@v` gemm are small. for such cases, does it really worth using Tensor Core for GEMM?